src/indexer.c

```diff
@@ -33,6 +34,39 @@ void IndexInit() {
     tsLabelIndex = RedisModule_CreateDict(NULL);
 }
 
+static void *NullDefrag(__unused RedisModuleDefragCtx *ctx,
+                        void *data,
+                        __unused unsigned char *key,
+                        __unused size_t keylen) {
+    return data;
+}
+
+static void *DefragIndexLeaf(RedisModuleDefragCtx *ctx,
+                             void *data,
+                             __unused unsigned char *key,
+                             __unused size_t keylen) {
+    static RedisModuleString *seekTo = NULL;
+    return (void *)defragDict(ctx, (RedisModuleDict *)data, NullDefrag, &seekTo);
```

Comment:
So the value is another dictionary? What happened if this inner dictionary will stop in the middle. I believe the defrag will assume it finished no and will not return to it no?